libvcx/src/api_lib/api_c/credential_def.rs

```diff
@@ -47,14 +47,15 @@ use crate::error::prelude::*;
 /// #Returns
 /// Error code as a u32
 #[no_mangle]
+#[deprecated(since = "0.25.0", note = "This function was split into vcx_credentialdef_generate_and_store and vcx_credentialdef_publish to allow tails_url customization")]
 pub extern fn vcx_credentialdef_create(command_handle: CommandHandle,
                                        source_id: *const c_char,
                                        credentialdef_name: *const c_char,
                                        schema_id: *const c_char,
                                        issuer_did: *const c_char,
                                        tag: *const c_char,
                                        revocation_details: *const c_char,
-                                       _payment_handle: u32,
+                                       tails_url: *const c_char,
```

Comment:
If we can't avoid making a breaking change on the old function, maybe we should just remove it instead of deprecating. 
I am okay with removing it, the migration is quite straightforward - call the two new functions in row instead.

Reply:
Alright